src/quickgui/plugin/qgsquickpositionmarker.qml

```diff
@@ -73,6 +75,24 @@ Item {
         id: positionKit
     }
 
+    Rectangle {
+        id: accuracyIndicator
+        visible: withAccuracy
+        x: positionMarker.screenPosition.x - width/2
+        y: positionMarker.screenPosition.y - height/2
+        width: {
+            if (positionKit.accuracy > 0)
+                QgsQuick.Utils.distanceToMapUnits(mapSettings, positionKit.accuracy, 0, 0, 0) * 2
```

Comment:
shouldn't be `(accuracyIndicator.x + positionKit.accuracy, accuracyIndicator.y, accuracyIndicator.x + positionKit.accuracy, accuracyIndicator.y)`?